### ISSUE TITLE
Callouts not being exported correctly

### ISSUE THREAD

**Issue by agnoldo** [OliverBalfour/obsidian-pandoc#145]
First of all, many thanks for this amazing plugin. I use it with Eisvogel template to generate PDF reports and the results are very good.

Unfortunately I tried to export Obsidian callouts and they didn't appear correctly when exporting to PDF via Pandoc. If I use Obsidian's built-in PDF export, then it works well, but Pandoc is much nicer for other things so I'd like to keep using it...

**Comment by aravindk100** [comment]
Yes I noticed the call outs are being split into two lines and not as rendered in Obsidian.

**Comment by manuelhubacher** [comment]
The same here. As far as I understand it, Pandoc uses its own [Markdown Syntax](https://pandoc.org/MANUAL.html#pandocs-markdown) that does not support Obsidian's callout blocks. They way to go is to use a Lua filter as [saf-dmitry](https://forum.obsidian.md/t/rendering-callouts-similarly-in-pandoc/40020/6) mentioned in the Obsidian Forum and style the resulting Pandoc’s native Divs so that they look similar to Obsidian’s callouts.
